src/trace_processor/storage/stats.h

```diff
@@ -406,58 +406,58 @@ namespace perfetto::trace_processor::stats {
   F(v8_code_load_missing_code_range,      kSingle,  kError,    kAnalysis,      \
       "V8 load had no code range or an empty one. Event ignored."),            \
   F(winscope_inputmethod_clients_parse_errors,                                 \
-                                          kSingle,  kInfo,     kAnalysis,      \
+                                          kSingle,  kDataLoss, kAnalysis,      \
```

Comment:
kDataLoss means the data is not in the trace because it could not be captured. Reading the description, it feels like all of these are errors not losses.

Reply:
Ah ok, then yeah, should be kError. Will update.